pages/linux/tree.md

```diff
@@ -10,25 +11,25 @@
 
 `tree -d`
 
-- Print hidden files too:
+- Print hidden files too with colorization on:
 
-`tree -a`
+`tree -a -C`
 
 - Print the tree without indentation lines, showing the full path instead (use `-N` to not escape whitespace and special characters):
 
 `tree -i -f`
 
-- Print the size of each node next to it, in human-readable format:
+- Print the size of each node next to it, in human-readable format, with directories displaying their cumulative size (as in the `du` command):
 
-`tree -s -h`
+`tree -s -h --du`
 
-- Filter the tree using a wildcard (glob) pattern:
+- Print files within the tree hierarchy, using a wildcard (glob) pattern, and pruning out directories that don't contain matching files:
 
-`tree -P {{*.txt}}`
+`tree -P '{{*.txt}}' --prune`
 
-- Ignore entries that match a wildcard (glob) pattern:
+- Print directories within the tree hierarchy, using the wildcard (glob) pattern, and pruning out directories that aren't ancestors of the wanted one:
```

Comment:
I understand you wanting to add more command line options, to this page, but this is rather advanced usage. Think of it this way: is this in the top 8 most common usages of `tree` you can think of? I would suggest keeping the original, or at best elaborating it while still keeping it simple.

Reply:
I'm just syncing this page with https://github.com/navarroaxel/tldr/blob/master/pages/osx/tree.md
Do you think we should shrink the osx page too?

Reply:
@navarroaxel okay, that's a good point, thank you for linking. I see that the OSX page has different because the command actually supports different options (see [relevant commit](https://github.com/tldr-pages/tldr/commit/31a07858ca2d010625499fb9e942f9c1371b339a) where this was introduced). Is this example compatible with the Linux version of the tool? If yes, then you are right that it makes sense to sync them, and we can work on a PR to simplify both later.

Reply:
Yeah, I've checked and the same options applies to the Linux `tree`, and all these arguments are explained in the `more information` link in this page.

Reply:
@mebeim Yeah, I've checked and the same options applies to the Linux `tree`, and all these arguments are explained in the `more information` link in this page, the same link than `osx` page.

Reply:
@navarroaxel thanks a lot